common/src/main/resources/less/style.less

```diff
@@ -409,3 +409,48 @@ body {
   .error-label-mixin();
 }
 
+
+.partner-split-data {
+  width: 6.5em;
+}
+
+.explore-text-in-form {
+  margin-top: auto;
+  margin-bottom: 3px;
```

Comment:
I wonder if we should make all margins be defined in `em`. I don't think there is a hard rule though.

Reply:
I got this number empirically in the dev tools. But, you're right, this should be in `form.overrides` and use the appropriate less variables to make this more generally usable, for example in forms of different "sizes".